src/utils/api-request.ts

```diff
@@ -1108,6 +1112,14 @@ export class AuthorizedHttp2Client extends Http2Client {
       const authHeader = 'Authorization';
       requestCopy.headers[authHeader] = `Bearer ${token}`;
 
+      // Fix issue where firebase-admin does not specify quota project that is
+      // necessary for use when utilizing human account with ADC (RSDF)
+      if (!requestCopy.headers['x-goog-user-project'] && this.app.options.projectId) {
+        requestCopy.headers['x-goog-user-project'] = this.app.options.projectId
+      }
```

Comment:
This is a bug in the current SDK and it should only be set for ADC human accounts. I have a fix coming up here https://github.com/firebase/firebase-admin-node/pull/2761

You can remove this part from this PR for now

Reply:
Done!